src/seedu/addressbook/commands/ReviseCommand.java

```diff
@@ -0,0 +1,59 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.Tag;
+
+import java.util.HashSet;
+import java.util.Set;
+
+
+/**
+ * Shows details of the person identified using the last displayed index.
```

Comment:
You can think about how this javadoc comment can be improved to better reflect the functionality of this command/class.